lib/common/common.go

```diff
@@ -60,3 +63,21 @@ func ParseDockerURL(arg string) (*ParsedDockerURL, error) {
 	p, err := docker.ParseDockerURL(arg)
 	return (*ParsedDockerURL)(p), err
 }
+
+// IsValidLayerRefs validates a layer refs
+func IsValidLayerRefs(name string) error {
+	validName := regexp.MustCompile(`^([A-Fa-f0-9_+.-]+)$`)
+	if ok := validName.MatchString(name); !ok {
+		return fmt.Errorf("invalid layer name %q", name)
+	}
+	return nil
+}
+
+// IsValidLayerId validates a layer ID
```

Comment:
My bad, this was just returning a bool at first but then changed to error returning, but I forgot to reflect that in the naming.